FirebaseAuth/Sources/Swift/Backend/RPC/SendVerificationTokenResponse.swift

```diff
@@ -15,9 +15,12 @@
 import Foundation
 
 struct SendVerificationCodeResponse: AuthRPCResponse {
-  var verificationID: String?
+  let verificationID: String
 
   init(dictionary: [String: AnyHashable]) throws {
-    verificationID = dictionary["sessionInfo"] as? String
+    guard let verificationID = dictionary["sessionInfo"] as? String else {
+      throw AuthErrorUtils.unexpectedResponse(deserializedResponse: dictionary)
```

Comment:
Ideally, this should be non-optional. I'm adding the error for cases where it's not. Are there success cases where verificationID _should be_ `nil`?